lib/services/runtime/utils.ts

```diff
@@ -168,13 +168,17 @@ export const addButtonsIfExists = <N extends BaseRequest.NodeButton>(
 
 export const getReadableConfidence = (confidence?: number): string => ((confidence ?? 1) * 100).toFixed(2);
 
+export const getGlobalNoMatch = (runtime: Runtime) => {
+  return runtime.version?.platformData.settings?.globalNoMatch;
+};
+
 export const getGenericGlobalNoMatchPrompt =
   ({ isPrompt }: { isPrompt: (prompt: unknown) => prompt is Prompt }) =>
   (runtime: Runtime) => {
-    const { version } = runtime;
-    const prompt = version?.platformData.settings?.globalNoMatch?.prompt;
-    return prompt && isPrompt(prompt) ? prompt : null;
+    const noMatch = getGlobalNoMatch(runtime);
+    return noMatch?.prompt && isPrompt(noMatch?.prompt) ? noMatch?.prompt : null;
```

Comment:
Oh, I missed that in the past, but the nullish coalescing is not needed after we check the presence of noMatch.

```suggestion
    return noMatch?.prompt && isPrompt(noMatch.prompt) ? noMatch.prompt : null;
```